web/src/markdown_config.ts

```diff
@@ -31,55 +30,6 @@ import {user_settings} from "./user_settings";
     when the lookups fail.
 */
 
-// TODO/typescript: Move this to markdown
```

Comment:
Can have a prep commit where this is removed. It didn't seem to be adding anything, and was making typescript harder